### PR TITLE
End WEIS Branch

### PR DESCRIPTION
**Pull request by dzalkind**
Check for `weis.aeroelasticse` and use `ofTools` if it's not available.

### CODE REVIEW

File: ROSCO_toolbox/controller.py [hunk 1]
```diff
@@ -515,9 +515,9 @@ def peak_shaving(self,controller, turbine):
     def min_pitch_saturation(self, controller, turbine):
         
         # Find TSR associated with minimum rotor speed
-        TSR_at_minspeed = (controller.pc_minspd/turbine.Ng) * turbine.rotor_radius / controller.v_below_rated
```
Comment (dzalkind) [inline ROSCO_toolbox/controller.py:518]: @nikhar-abbas, I think we want to revert this to match `develop` but I don't remember the history here.  What should it be?

Comment (nikhar-abbas) [reply ROSCO_toolbox/controller.py:518]: I don't quite remember the history here either. We're defining `controller.pc_minspd = controller.vs_minspd` [here](https://github.com/NREL/ROSCO_toolbox/blob/2cc92c9a4c11545df023cde81771787d0e962375/ROSCO_toolbox/controller.py#L284), though this probably redundant and unnecessary. 

So, I like the change you made here - maybe just delete the line linked above and it should be good.

**Review by nikhar-abbas — CHANGES_REQUESTED** [review]
Once the `pc_minspd` stuff is all deleted and converted to `vs_minspd`, I think this is ready to go!

**Comment by dzalkind** [comment]
I'm reverting that min pitch setting to what it is in `develop`.  It's a pretty minor change that says we'll set a min pitch when the TSR is greater than the `TSR_operational` set in the yaml.  In the `weis` branch, you had changed that to set a min pitch when the TSR was greater than the optimal TSR as determined by the Cp surface.  If we don't set a `TSR_operational` in the yaml, it uses the optimal Cp from the surface. 

I think this is good to go now.

**Pull request merged**